tests/share/normalize/test_links.py

```diff
@@ -1,8 +1,35 @@
 import pytest
 import rfc3987
 
-from share.normalize.links import DOILink
-from share.normalize.links import OrcidLink
+import calendar
+
+from share.normalize.links import DOILink, OrcidLink, DateParserLink
+
+
+@pytest.mark.parametrize('date, result', [
+    (None, None),
```

Comment:
`None` should raise an exception of some kind. We'll want to know if a normalizer is unknowingly passing along `None` values.